hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -1190,28 +1193,54 @@ private PersistedUserVolumeInfo getVolumesByUser(String userNameKey)
   }
 
   @Override
-  public List<String> getExpiredOpenKeys(Duration expireThreshold,
-      int count) throws IOException {
+  public List<OmOpenKeyInfo> getExpiredOpenKeys(Duration expireThreshold,
+      int limit) throws IOException {
+    BucketLayout first, second;
+    if (random.nextBoolean()) {
```

Comment:
@kaijchen Nice logic. Please add comment to explain it.

Reply:
This might be changed to 2 different methods as discussion in #3235.
If we are still using this logic, I will add javadoc to explain it.

Reply:
I could see three patches( #3226 [3226](https://github.com/apache/ozone/pull/3226), [#3233](https://github.com/apache/ozone/pull/3233), and [#3235](https://github.com/apache/ozone/pull/3235)) and it would be good if you could tell me the order of these patches. That would help me to do create my review pipeline first->second->third :-)

Reply:
#3226 depends on #3235.
#3233 is independent.

Reply:
OK, thanks!